config/crd/kustomization.yaml

```diff
@@ -6,6 +6,7 @@ resources:
 - bases/opentelemetry.io_instrumentations.yaml
 - bases/opentelemetry.io_opampbridges.yaml
 - bases/opentelemetry.io_targetallocators.yaml
+- bases/opentelemetry.io_clusterobservabilities.yaml
```

Comment:
Adding the CRD here will cause it to be shipped in our release manifest, which I'd prefer to avoid for now. For testing, we can include it manually.